src/ui/noscrollcombobox.cpp

```diff
@@ -16,6 +16,10 @@ NoScrollComboBox::NoScrollComboBox(QWidget *parent)
     setEditable(true);// can set to false manually when using
     this->completer()->setCompletionMode(QCompleter::PopupCompletion);
     this->completer()->setFilterMode(Qt::MatchContains);
+
+    QRegularExpression re("[A-Za-z_0-9]+");
```

Comment:
I'm worried this regex is too restrictive.  Perhaps it would be better to just exclude whitespace?  I'm not sure.

Reply:
it seems like these are all labels, constants, or numbers, but it is probably better to be cautious here I could me missing something

Reply:
although you'd still be able to type '\n' into the box, you can't paste any spaces